### PR TITLE
Sankofa Display: Version 1.000; ttfautohint (v1.8.4.7-5d5b) added



### LINKED ISSUE #7192: Add Sankofa Display
[**Sankofa Display Upstream**](https://github.com/batsimadz/Sankofa-Display)

**Super short description of the Font Family:**
Sankofa Display is a typeface that draws inspiration from African art styles, with a focus on straight-line geometric designs.

**Requirements:**

Google Fonts will publish only fonts that match its requirements. Please familiarize yourself with the complete documentation in the [Google Fonts Guide](https://googlefonts.github.io/gf-guide/) (GF-Guide) and ensure your font project complies with them before submitting the font family. You can also use the [Google Fonts Project Template](https://github.com/googlefonts/googlefonts-project-template), which will help you create a repository that follows the needed structure and includes build requirements.

By filling this issue, you can confirm the project meets the  requirements (by ticking the cases or putting x between the square brackets in text mode):

- [x] I read, agree, and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/index#pre-production-getting-your-fonts-ready-for-gf)
- [x] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
- [x] There is no "Reserved Font Name" mention in the OFL license
- [x] Family name is unique according to [namecheck.fontdata.com](https://namecheck.fontdata.com/)
- [x] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
- [x] The repository respect the [upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
- [x] The source files are available in the repository
- [x] I am the author of the font, or the author agreed to this submission
- [x] I will maintain the repository and participate in the onboarding process (addressing, solving, and responding to issues, merging pull requests, etc)

**Image:**
sankofa-cover

### PR DESCRIPTION
**Pull request by m4rc1e**
Taken from the upstream repo https://www.github.com/batsimadz/Sankofa-Display at commit https://www.github.com/batsimadz/Sankofa-Display/commit/ee4cefac12dd65aff8be308854b717f374a2d89b.
## PR Checklist:

- [x] Family categorization tags collected from the type design team with the Categories Form
- [x] Minisite_url definition in the METADATA.pb file for commissioned projects
- [x] Primary_script definition in the METADATA.pb file for all projects that have a primary non-Latin based language support target
- [x] Fontbakery checks are reviewed and failing checks are resolved in collaboration with the upstream font development team
- [x] Diffenator2 regression checks for revisions on all projects that are currently in production
- [x] Designers bio info have to be present in the designer catalog (at least an issue should be opened for tracking this, if they are not)
- [x] Check designers order in metadata.pb, since the first one of the list appears as “principal designer”
- [x] Social media formatted visual assets for all new commissioned projects in the Drive directory, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet
- [x] Social media content draft for all new commissioned projects in the Drive directory and Social Media tracker spreadsheet, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet

Fixes #7192

### CODE REVIEW

File: ofl/sankofadisplay/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1 @@
+N/A
```
Comment (m4rc1e) [inline ofl/sankofadisplay/DESCRIPTION.en_us.html:1]: @n8willis Do we have a description for the family?

Comment (n8willis) [reply ofl/sankofadisplay/DESCRIPTION.en_us.html]: Added. This is in the DESCRIPTION.en_us.html from upstream; not sure why it wasn't here.

**Comment by m4rc1e** [comment]
@simoncozens your beziers lib is producing a ZeroDivisionError. Here's the traceback from the report:

```
Failed with ZeroDivisionError: float division by zero

  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/fontbakery/checkrunner.py", line 213, in _run_check
    subresults = list(subresults)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/fontbakery/checks/outline.py", line 366, in com_google_fonts_check_outline_direction
    if path.direction == 1:
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/beziers/path/__init__.py", line 558, in direction
    return math.copysign(1, self.signed_area)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/beziers/path/__init__.py", line 541, in signed_area
    flat = self.flatten()
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/beziers/path/__init__.py", line 494, in flatten
    segs.extend(s.flatten(degree))
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/beziers/quadraticbezier.py", line 58, in flatten
    samples = self.sample(self.length/degree)
  File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/beziers/utils/samplemixin.py", line 20, in sample
    step = 1.0 / float(samples)
```

**Comment by simoncozens** [comment]
This is fonttools/fontbakery#4699, I'll fix it.

**Comment by simoncozens** [comment]
Should be fixed in beziers 0.6.0, just uploaded.

**Comment by n8willis** [comment]
Is there anything I can do to prep this between now and the Fontbakery PR landing?

**Comment by vv-monsalve** [comment]
@n8willis Inspecting the font in dev-server, Batsirai Madzonga designer bio information still needs to be added.

**Pull request merged**